test/plain-list/AutosuggestApp.js

```diff
@@ -54,6 +54,18 @@ export const onSuggestionsUpdateRequested = sinon.spy(({ value }) => {
   });
 });
 
+export const tabToSelect = {
```

Comment:
Note: also going to update this API per the comments [here](https://github.com/moroshko/react-autosuggest/pull/165#discussion_r62430407).